blockly-core/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+cd tests
+
+# Make sure phantomjs 2+ is installed
+phantomjsversion=$(node_modules/phantomjs/bin/phantomjs --version)
+if [ $phantomjsversion != "2.0.0" ]; then
+  npm install
```

Comment:
I'm trying to decide whether I like the idea of running npm install here directly vs. outputting a message telling the user they should do that. It seems useful this way, but in our other projects there aren't any instances of doing something like this.